samples/apps/copilot-chat-app/webapi/CopilotChat/Options/PlannerOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SemanticKernel.Service.CopilotChat.Options;
+
+/// <summary>
+/// Configuration options for the planner.
+/// </summary>
+public class PlannerOptions
+{
+    public const string PropertyName = "Planner";
+
+    /// <summary>
+    /// Define if the planner must be Sequential or not.
+    /// </summary>
+    public bool IsSequential { get; set; } = false;
```

Comment:
Could we consider using an enum/free value like we do for MemoriesStore and ChatStore? 
```
"Planner": {
  "Type": "Sequential"
}
```

Reply:
Done with a free variable